tests/thrift_dicts.py

```diff
@@ -34,6 +34,24 @@
         },
         "processes": [
             {
+                "name": "mkdir",
+                "cmdline": 'mkdir -p /mnt/mesos/sandbox/sandbox/__jobio/input /mnt/mesos/sandbox/sandbox/__jobio/output',
+                "final": False, "daemon": False, "ephemeral": False,
+                "max_failures": 1, "min_duration": 5
+            },
+            {
+                "name": "symlink_in",
+                "cmdline": 'ln -s /mnt/mesos/sandbox/sandbox/__jobio/input /job/input',
+                "final": False, "daemon": False, "ephemeral": False,
+                "max_failures": 1, "min_duration": 5
+            },
+            {
+                "name": "symlink_out",
+                "cmdline": 'ln -s /mnt/mesos/sandbox/sandbox/__jobio/output /job/output',
+                "final": False, "daemon": False, "ephemeral": False,
+                "max_failures": 1, "min_duration": 5
+            },
```

Comment:
It seems unfortunate that essentially the same command line has to be maintained in four places (cli/thrift and production/test).  Would it make sense to refactor this to use a single representation of a command line?